prow/cmd/entrypoint/BUILD.bazel

```diff
@@ -14,6 +14,26 @@ go_library(
     ],
 )
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+container_image(
+    name = "base-image",
+    # TODO: this base image is *different* to the one defined in the Dockerfile?
+    base = "@git-base//image",
```

Comment:
I noticed the Dockerfile specifies an alpine base, but the Bazel build file previously specified git. I have opted to continue using the bazel-defined base here and left the Dockerfile as-is.